### DISCUSSION TITLE
Send weekly reports to a proton.me email address ?

### DISCUSSION THREAD

**Discussion by Fish31520** [Nachtzuster/BirdNET-Pi#203] [Q&A]
Hi folks,

I'm currently trying to de-Google my devices. I've switched to Proton Mail but I can't get BirdnetPi to send me weekly reports on my new address.

Is there a way to do that ?

Thanks !

**Comment by alexbelgium** [comment]
Probably through apprise indeed, you should have a look at their documentation !

You could also have a look on the old repo, there might be some indications on how people got emails to work ;-) At a glance, I saw for example https://github.com/mcguirepr89/BirdNET-Pi/discussions/630

Hopefully someone also using proton mail would come along and have more precise suggestions for you !

**Comment by Emmo213** [comment]
It does use the apprise.txt configuration. What does yours look like? Do you have a mailto: line configured to your new proton email address?

**Reply by alexbelgium** [reply]
I only use telegram so can't really help you... But I hope people at apprise's github will help you, I see you have contacted them

**Reply by Fish31520** [reply]
@Emmo213 

Hi,

My apprise.txt file only contains `mailto://user:password@proton.me`

@alexbelgium 

Salut,

I'll let you guys know if I find any answer from apprise's github. Could be usefull to others...

Bye !

Jean

**Reply by Emmo213** [reply]
@Fish31520 are you using a free Proton account or do you have a paid version?

**Reply by Fish31520** [reply]
I gave a try to the free version... 😅

**Reply by Emmo213** [reply]
> I gave a try to the free version... 😅

According to their documentation you need to have a paid version to support SMTP.

**Reply by Fish31520** [reply]
@Emmo213 

😮 Seems like you're right ! I didn't know a premium Proton account was necessary for notifications ! 

Anyway thanks ! 😄
